app/utils/jwt.js

```diff
@@ -7,7 +7,7 @@ export default Ember.Object.extend({
   payload: function(){
     var token = this.get('token');
 
-    if (!token) { return {}; }
+    if (!token || token === "null") { return {}; }
```

Comment:
I'm not sure- bad data gets written at some point. For all I know it was a test or a half-complete implementation of something reloading with live-reload. IMO it seems good to catch when possible.